src/templates/_partials/action.twig

```diff
@@ -1,4 +1,4 @@
-<form method="post" accept-charset="UTF-8">
+<form style="margin-bottom:4px;" method="post" accept-charset="UTF-8">
```

Comment:
Gives the extra buttons room to breath on smaller viewports:

![After](https://user-images.githubusercontent.com/1508604/132098553-87145fc1-5919-457d-9299-4a3f3d50e230.jpg)
vs.
![After](https://user-images.githubusercontent.com/1508604/132098573-677b0fac-dd58-4eef-a090-fda7931fc98b.jpg)